.github/pull_request_template.md

```diff
@@ -6,6 +6,18 @@ feat(validatorName): brief title of what has been done
 <!--- briefly describe what you have done in this PR --->
 
 ## Checklist
+Validator added:
```

Comment:
Can you discard your changes to this file? 
I suppose you made a copy/paste error

Reply:
Thanks for your help.  I'm not sure what it is exactly you'd like me to delete, could you please point i

Reply:
Point it out, obviously I have no clue what I'm doing or how I wound up here. Sorry about that

Reply:
Yes Surely I do required changes

Reply:
What is your opinion about isCountryByCode  i could change to isvalidCountryCode ?

My opinion about isValidCountryName= it will beneficial when we are not providing those search country option